webapp/pages/profile/_id/_slug.vue

```diff
@@ -375,14 +375,14 @@ export default {
     showMoreContributions() {
       const { Post: PostQuery } = this.$apollo.queries
       if (!PostQuery) return // seems this can be undefined on subpages
-
       this.offset += this.pageSize
+
       PostQuery.fetchMore({
         variables: {
           offset: this.offset,
           filter: this.filter,
           first: this.pageSize,
-          orderBy: this.sorting,
+          orderBy: 'createdAt_desc',
```

Comment:
Ouch! So `this.sorting` was always null? Is this the bug?

Reply:
I feel like this could have been related to the bug, especially since I haven't been able to reproduce it after I changed this, but we would need to test it further, I guess... maybe on `nitro-staging`?